docs/src/generate_bls_to_execution_change.md

```diff
@@ -0,0 +1,30 @@
+# generate-bls-to-execution-change
+
+## Description
+Generates a BLS to execution address change message.
+
+## Optional Arguments
+
+- **`--bls_to_execution_changes_folder`**: The path to store the change JSON file.
+
+- **`--chain`**: The chain to use for generating the deposit data. Options are: 'mainnet', 'holesky', etc.
+
+- **`--mnemonic`**: The mnemonic you used to create withdrawal credentials.
+
+- **`--mnemonic_password`**: The mnemonic password you used in your key generation. Note: It's not the keystore password.
+
+- **`--validator_start_index`**: The index position for the keys to start generating withdrawal credentials for.
+
+- **`--validator_indices`**: A list of the chosen validator index number(s) as identified on the beacon chain. Split multiple items with whitespaces or commas.
+
+- **`--bls_withdrawal_credentials_list`**: A list of the old BLS withdrawal credentials of the given validator(s). It is for confirming you are using the correct keys. Split multiple items with whitespaces or commas.
+
+- **`--execution_address`**: If this field is set and valid, the given Eth1 address will be used to create the withdrawal credentials. Otherwise, it will generate withdrawal credentials with the mnemonic-derived withdrawal public key.
```

Comment:
Same as the other Eth 1 issue. Related to #29

Reply:
ack